Python/pywarpx/_libwarpx.py

```diff
@@ -1709,6 +1769,41 @@ def get_mesh_charge_density_fp_lovects(level, include_ghosts=True):
     '''
     return _get_mesh_array_lovects(level, None, include_ghosts, libwarpx.warpx_getChargeDensityFPLoVects)
 
+def get_num_particles_impacted_boundary(species_name, boundary):
+    '''
+
+    This returns the number of particles that have been scraped so far in the simulation
+    from the specified boundary and of the specified species.
+
+    Parameters
+    ----------
+
+        species_name   : return the number of scraped particles of this species
+        boundary       : the boundary from which to get the scraped particle data.
+                         In the form x/y/z_hi/lo
+
+    Returns
+    -------
+
+        The number of particles scraped so far from a boundary and of a species.
+
+    '''
+    if geometry_dim == '3d':
+        dimensions = {'x' : 0, 'y' : 1, 'z' : 2}
```

Comment:
[Earlier in `get_particle_boundary_buffer`,](https://github.com/ModernElectron/WarpX/pull/48/files#diff-45fe8ad0cfda5ed498793502822a96a08d19326c06d5a73b5f876f37c0568364R586) the dimensions are set as `{'x' : 1, 'y' : 2, 'z' : 3}` wheres here, the dimensions are set as `{'x' : 0, 'y' : 1, 'z' : 2}`. Is this intentional?

Reply:
Yeah that was an error. It should be 0, 1, 2.